src/components/lyne-tab-group/lyne-tab-group.tsx

```diff
@@ -0,0 +1,284 @@
+      if (this.tabs[this.selectedIndex].hasAttribute('disabled')) {
+        this._getEnabledTabs()[0]?.tabGroupActions.toggle();
+      } else {
+        this.tabs[this.selectedIndex].tabGroupActions.toggle();
+      }
+    } else {
+      this._getEnabledTabs()[0]?.tabGroupActions.toggle();
+    }
+  }
+
+  private _onTabAttributesChange(mutationsList): void {
+    for (const mutation of mutationsList) {
+      const tab = (mutation.target as InterfaceLyneTabGroupTab);
+
+      if (mutation.type === 'attributes') {
+        if (mutation.attributeName === 'disabled') {
+          if (this._isValidTabAttribute(tab, 'disabled')) {
+            tab.tabGroupActions.disable();
+          } else {
+            tab.tabGroupActions.enable();
+          }
+        }
+        if (mutation.attributeName === 'active') {
+          if (this._isValidTabAttribute(tab, 'active') && !tab.disabled) {
+            tab.tabGroupActions.toggle();
+          } else {
+            tab.tabGroupActions.deactivate();
+          }
+        }
+      }
+    }
+  }
+
+  private _isValidTabAttribute(tab: InterfaceLyneTabGroupTab, attribute: string): boolean {
+    return tab.hasAttribute(attribute) && tab.getAttribute(attribute) !== 'false';
+  }
+
+  private _configure(tab: InterfaceLyneTabGroupTab): void {
+    tab.tabGroupActions = {
+      activate: (): void => {
+        tab.setAttribute('active', '');
+        tab.active = true;
+        tab.tabIndex = 0;
+        tab.setAttribute('aria-selected', 'true');
+        tab.relatedContent?.setAttribute('active', '');
+      },
+      deactivate: (): void => {
+        tab.removeAttribute('active');
+        tab.active = false;
+        tab.tabIndex = -1;
+        tab.setAttribute('aria-selected', 'false');
+        tab.relatedContent?.removeAttribute('active');
+      },
+      disable: (): void => {
+        if (!tab.disabled) {
+          if (!tab.hasAttribute('disabled')) {
+            tab.setAttribute('disabled', '');
+          }
+          tab.disabled = true;
+          tab.tabIndex = -1;
+          tab.setAttribute('aria-selected', 'false');
+          tab.relatedContent?.removeAttribute('active');
+
+          if (tab.active) {
+            tab.removeAttribute('active');
+            tab.active = false;
+            this._getEnabledTabs()[0]?.tabGroupActions.toggle();
+          }
+        }
+      },
+      enable: (): void => {
+        if (tab.disabled) {
+          tab.removeAttribute('disabled');
+          tab.disabled = false;
+        }
+      },
+      toggle: (): void => {
+        if (!tab.active && !tab.disabled) {
+          this.tabs.find((t) => t.active)?.tabGroupActions.deactivate();
+          tab.tabGroupActions.activate();
+        } else if (tab.disabled) {
+          console.warn('You cannot activate a disabled tab');
+        }
+      }
+    };
+    this._ensureId(tab);
+    tab.relatedContent = tab.nextElementSibling?.tagName === 'DIV'
```

Comment:
Should not other elements as divs be supported?

Reply:
No, theoretically only the divs following the heading tags should represent related content...

Reply:
Definitely, I remember it was a topic of discussion